src/Chainweb/Pact/TransactionExec.hs

```diff
@@ -215,6 +216,9 @@ applyCmd v logger gasLogger pdbenv miner gasModel txCtx spv cmd initialGas mcach
       cr <- catchesPactError $! runPayload cmd managedNamespacePolicy
       case cr of
         Left e
+          | chainweb217Pact' -> do
+            r <- jsonErrorResult (PactError ArgsError def [] "") "tx failure for request key when running cmd"
```

Comment:
Would this also swallow error messages from local results?

If that's the case, why not leave the message untouched here and only drop it when assembling the final result for PayloadWithOutputs and computing the PayloadHash?

Reply:
@larskuhtz this is a different code path than local cmd execution, which is done in `applyLocal`.

Reply:
What about newBlock? I assume it is using this code path?

Reply:
I believe so yeah

Reply:
> If that's the case, why not leave the message untouched here and only drop it when assembling the final result for PayloadWithOutputs and computing the PayloadHash?

`jsonErrorResult` `show`s the error, which fill force it and cause the long computation.